service_contracts.rst

```diff
@@ -31,6 +31,109 @@ ServiceRegistry
 
 The ServiceRegistry provides a registry in which services have to register before becoming a full part of the Raiden services system. Services have to deposit RDN tokens in this contract for a successful registration. This avoids attacks using a large number of services and increases the incentive for service provider to not harm the Raiden ecosystem.
 
+Requirements
+------------
+
+- Need to stake for these slots by the Raiden Service Providers (RSPs), but there will be no slashing
+- RSPs must be allowed to withdraw the deposited stake after their registration ends
+- An "auction" mechanism that allows for competition over those slots
+- There must be a way for new RSPs to get a slot
+- We want an easy MVP to get real user feedback after our release
+- Need to provide a way of setting the minimal amount of deposits - the cost of becoming an RSP should be high enough to deter users from double-spending IOUs
```

Comment:
```suggestion
- Need to provide a way of setting the minimal deposit amount - the cost of becoming an RSP should be high enough to deter users from double-spending IOUs
```

Otherwise it might be confused with number of deposits.